ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -27,11 +27,84 @@
                         Search by NTBS, ETS, LTBR or NHS number
                     </span>
                     <span ref="errorField" nhs-span-type="ErrorMessage" id="id-error"
-                        asp-validation-for="SearchParameters.IdFilter"></span>
+                        asp-validation-for="SearchParameters.IdFilter" >
+                    </span>
                     <input is-error-input=@IdFilterError nhs-input-type="Standard"
                         asp-for="SearchParameters.IdFilter" classes="nhsuk-input--width-10" />
                 </nhs-form-group>
             </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                @{
+                    var DobError = !Model.validationService.IsValid("SearchParameters.PartialDob");
+                    var DobGroupState = DobError ? Error : Standard;
+                }
+                <nhs-form-group nhs-form-group-type=@DobGroupState>
+                    <label nhs-label-type="Standard"> Date of birth </label>
+                    <span class="nhsuk-hint " id="example-hint">

```

Comment:
there should be a hint tag helper? Need better id, too, and I think we want to reference the id in some `aria-described-by` attribute on the `fieldset`: https://design-system.service.gov.uk/components/date-input/#default-2-open-html